### DISCUSSION TITLE
Search through multiple editors

### DISCUSSION THREAD

**Discussion by pouqwyggad** [TypeFox/monaco-languageclient#715] [Q&A]
Hi! I ran into a problem, in each instance of monaco, there is a search via ctrl + f, I have a component that contains the editors of Monaco, it turns out like .ipynb, can I take the search to the level of the component that contains these editors, and not in the editors themselves. But at the same time, so that it is the functionality that the monaco editor itself provides.

https://microsoft.github.io/monaco-editor/playground.html?source=v0.50.0#XQAAAAJqBAAAAAAAAABBqQkHQ5NjdMjwa-jY7SIQ9S7DNlzs5W-mwj0fe1ZCDRFc9ws9XQE0SJE1jc2VKxhaLFIw9vEWSxW3yscw6ncjO_1uiCQxvQ4mP-qO7F7UtcAIxCG82YQcF_a0oso9lRXYgIdjkxyPDKD2h3_Ie8hOabMZb3NoFKe2zeWAPvxhaC_umu42UTCNoy5sELJxnAAUSXtqUaFMqqzd6QHRfOarwLHO7YmSy4rdPveLQppjkOZteF4gZzsPBZh39VFiEYuKcrJLpyfaCGi6zpQ_pt32I9i125YdxXkSnCFIIzQYb4aRCcM8p38Z7PXPD6NTs8Ub-c2Y0htmXjLW6A89xEXqdo3XxzRfyzMcVNZU2pGAJ2jZruQThM-xOBVK2jciPJ9S-9o1JJedUPFFBT5JwtDbUNig2vffCsCSYOQXrIsCz9e-Ih1ep7oP_KDp70HPq3wgsGAqHaQvOH6tBGycn7ERMrjmIE25rmlDu6JJe3O7G5OlQcEKKjq6V_AJ8hveRksu-xfrUnCJnWle3Z7SkXwqNwpmhsUgUGk5tbwqGNM-Xh6FFpkDOdCEp0bFF9JGJQ6QZMbpqUl-o-QzpfHyLU04Tgw75UmoW9W3_A5srkP_IKpZAA

*the screenshot shows 2 editors and a container, but when you call the search on the second one, it appears on the second one, and when you click on the container (red border), a browser search is called. I expect the behavior as in our code, that when you click on a cell (editor), there will be a search throughout the container (conditionally for all editors inside)

![image](https://github.com/user-attachments/assets/ba4e41e0-98e2-4a01-bbf0-022f324f62f6)

I expect behavior like in vscode
![image](https://github.com/user-attachments/assets/ed003cc4-7be0-4f0f-96a5-0d7a39192611)

I would appreciate any help, thank you!!

**Comment by CGNonofr** [comment]
I'm not aware of any simple solution. What you expect is the behavior of VSCode notebook editor pane, which has probably some logic to do that, you'll either need to use the VSCode notebook component or re-implement it yourself
